code/go/0chain.net/blobber/main.go

```diff
@@ -1,491 +1,47 @@
-	reader.Close()
-
-	reader, err = os.Open(*minioFile)
-	if err != nil {
+	if err := setupNode(); err != nil {
+		logging.Logger.Error("Error setting up blobber node " + err.Error())
 		panic(err)
 	}
 
-	err = setupMinioConfig(reader)
-	if err != nil {
+	if err := setupServerChain(); err != nil {
+		logging.Logger.Error("Error setting up server chain" + err.Error())
 		panic(err)
 	}
-	reader.Close()
-
-	node.Self.SetKeys(publicKey, privateKey)
-
-	port, err := strconv.Atoi(*portString) //fmt.Sprintf(":%v", port) // node.Self.Port
-	if err != nil {
-		Logger.Panic("Port specified is not Int " + *portString)
-		return
-	}
-
-	node.Self.SetHostURL(*hostname, port)
-	Logger.Info(" Base URL" + node.Self.GetURLBase())
-
-	config.SetServerChainID(config.Configuration.ChainID)
 
-	common.SetupRootContext(node.GetNodeContext())
-	//ctx := common.GetRootContext()
-	serverChain = chain.NewChainFromConfig()
-
-	if node.Self.ID == "" {
-		Logger.Panic("node definition for self node doesn't exist")
-	} else {
-		Logger.Info("self identity", zap.Any("id", node.Self.ID))
+	if err := setupDatabase(); err != nil {
+		logging.Logger.Error("Error setting up data store" + err.Error())
+		panic(err)
 	}
 
-	initIntegrationsTests(node.Self.ID)
-
-	//address := publicIP + ":" + portString
-	address := ":" + *portString
-
-	chain.SetServerChain(serverChain)
-
-	setupDatabase()
-
 	// Initialize after server chain is setup.
-	if err := initEntities(); err != nil {
-		Logger.Error("Error setting up blobber on blockchian" + err.Error())
-	}
-	if err := setup(*logDir); err != nil {
-		Logger.Error("Error setting up blobber on blockchian" + err.Error())
-	}
-	mode := "main net"
-	if config.Development() {
-		mode = "development"
-	} else if config.TestNet() {
-		mode = "test net"
-	}
-	Logger.Info("Starting blobber", zap.Int("available_cpus", runtime.NumCPU()), zap.String("port", *portString), zap.String("chain_id", config.GetServerChainID()), zap.String("mode", mode))
-
-	var server *http.Server
-
-	r := mux.NewRouter()
-
-	common.ConfigRateLimits()
-	initHandlers(r)
-
-	if config.Development() {
-		// No WriteTimeout setup to enable pprof
-		server = &http.Server{
-			Addr:              address,
-			ReadHeaderTimeout: 30 * time.Second,
-			MaxHeaderBytes:    1 << 20,
-			Handler:           r,
-		}
-	} else {
-		server = &http.Server{
-			Addr:              address,
-			ReadHeaderTimeout: 30 * time.Second,
-			WriteTimeout:      30 * time.Second,
-			IdleTimeout:       30 * time.Second,
-			MaxHeaderBytes:    1 << 20,
-			Handler:           r,
-		}
+	if err := setupFileStore(); err != nil {
+		logging.Logger.Error("Error setting up file store" + err.Error())
+		panic(err)
 	}
-	common.HandleShutdown(server)
-	handler.HandleShutdown(common.GetRootContext())
-
-	Logger.Info("Ready to listen to the requests")
 
-	if config.Development() {
-		go startGRPCServer(*r, grpcPortString)
-	}
+	// only enabled with "// +build integration_tests"
+	initIntegrationsTests(node.Self.ID)
 
```

Comment:
Why did you remove the grpcServer setting up from the code?

Reply:
it is moved to `startHttpServer`, not removed.